src/_includes/index_cta.html

```diff
@@ -1,18 +1,88 @@
 <section class="modal" data-modal="one">
-  <h2 class="modal__title">
-    Schedule a Demo
-  </h2>
-  <button class="modal__close">{% svg assets/svg/x.svg %}</button>
-  <div class="modal__content">
-    <div class="calendly-inline-widget" data-url="https://calendly.com/dbentley/tilt-enterprise?hide_event_type_details=1">
+  <div class="calendly">
```

Comment:
One small nit: These classes are global, so we should add a prefix to `calendly` and `callback`...maybe the same `modal-` prefix?

Reply:
Yeah totally, will do.